perception/realsense_driver/python/realsense_wrapper/realsense_d435.py

```diff
@@ -51,12 +51,38 @@ def get_intrinsics(self):
     def get_num_cameras(self):
         return len(self.device_ls)
 
-    def get_images(self):
+    def get_images(self, depth=False):
         framesets = self._get_frames()
+
+        if depth:
+            align_to = rs.stream.color
+            align = rs.align(align_to)
+            framesets = [align.process(frameset) for frameset in framesets]
+
         imgs = []
+        depth_imgs = []
         for frameset in framesets:
             frame = frameset.get_color_frame()
             img = np.asanyarray(frame.get_data())
             imgs.append(img)
 
-        return imgs
+            if depth:
+                frame = frameset.get_depth_frame()
+                depth_img = np.asanyarray(frame.get_data())
+                depth_imgs.append(depth_img)
+
+        if depth:
+            return imgs, depth_imgs
+        else:
+            return imgs
+
+    def get_pointcloud(self) -> rs.pointcloud:
+        framesets = self._get_frames()
+        pc_ls = []
+        for frameset in framesets:
+            pc = rs.pointcloud()
+            pc.map_to(frameset.get_color_frame())
+            points = pc.calculate(frameset.get_depth_frame())
```

Comment:
should we use aligned depth frame here?

Reply:
In some cases we want the raw point cloud without interpolations, and in other cases we want to align the points within the point cloud to pixels in the rgb image. Since this is meant to be a general API, we should probably support both.

Reply:
Yes, probably should support both